README.md

```diff
@@ -3,6 +3,7 @@
 Below are the infrastructures we currently have best practices for. Navigate to each provider to see what will be provisioned.
 
 - [AWS](terraform/providers/aws/README.md)
+- [Google Cloud Engine](terraform/providers/gce/README.md)
```

Comment:
Nitpicky, but we should update this to be GCE, or AWS to be Amazon Web Services so they are consistent.